server/controllers/aws.controller.js

```diff
@@ -108,6 +108,11 @@ export function copyObjectInS3(req, res) {
   });
 }
 
+export function listObjectsInS3(req, res) {
+  req.params = Object.assign({}, req.user);
```

Comment:
i think overwriting req.params is confusing—maybe `listObjectsInS3ForUser` could check if req.params.username is defined, and if not, use req.user?

Reply:
done :)